src/openassetio-python/tests/package/hostApi/test_manager.py

```diff
@@ -1134,7 +1140,234 @@ def test_when_zero_pageSize_then_InputValidationException_is_raised(
             )
 
 
-class Test_Manager_getWithRelationships:
+class Test_Manager_getWithRelationship_singular_convenience:
+    @pytest.mark.parametrize(
+        "error_mode",
+        [
+            None,
+            Manager.BatchElementErrorPolicyTag.kException,
+            Manager.BatchElementErrorPolicyTag.kVariant,
+        ],
+    )
+    def test_when_success_then_entityReferencePager_returned(
+        self,
+        manager,
+        a_ref,
+        mock_manager_interface,
+        a_host_session,
+        an_empty_traitsdata,
+        an_entity_trait_set,
+        a_context,
+        mock_entity_reference_pager_interface,
+        invoke_getWithRelationship_success_cb,
+        error_mode,
+    ):
+        page_size = 3
+        method = mock_manager_interface.mock.getWithRelationship
+
+        def call_callbacks(*_args):
+            invoke_getWithRelationship_success_cb(0, mock_entity_reference_pager_interface)
+
+        method.side_effect = call_callbacks
+
+        args = {
+            "entityReference": a_ref,
+            "relationshipTraitsData": an_empty_traitsdata,
+            "pageSize": page_size,
+            "relationsAccess": access.RelationsAccess.kRead,
+            "context": a_context,
+            "resultTraitSet": an_entity_trait_set,
+        }
+
+        if error_mode is not None:
+            args["errorPolicyTag"] = error_mode
+
+        actual_pager = manager.getWithRelationship(**args)
+
+        method.assert_called_once_with(
+            [a_ref],
+            an_empty_traitsdata,
+            an_entity_trait_set,
+            page_size,
+            access.RelationsAccess.kRead,
+            a_context,
+            a_host_session,
+            mock.ANY,
+            mock.ANY,
+        )
+
+        assert isinstance(actual_pager, EntityReferencePager)
+        actual_pager.get()  # Assert that pager is kept alive.
```

Comment:
This was me trying to test more without massive duplication, essentially doing the `test_pager_kept_alive_by_retaining_shared_ptr` test. Think it's redundant for the conveniences? I figured since there's delegation between additional functions going on it might matter.

Reply:
Ah I see, thought it must be something like that. Unfortunately this wouldn't reveal any problems, since `mock_entity_reference_pager_interface` is still in scope in Python. 

The `test_pager_kept_alive_by_retaining_shared_ptr` test creates a pager that immediately goes out of scope in Python. But it's (hopefully) kept alive by the `PyRetainingSharedPtr` mechanism. That's what the test is asserting.

As for whether we need to test this in the conveniences too... I had to refresh my memory but in short, no I don't think we do.  It's just checking the Python bindings of the `ManagerInterface` success callback, which is already independently tested.

So I think we can simply remove these calls.

Reply:
:+1: 

There was only one left after all the other changes anyhow. https://github.com/OpenAssetIO/OpenAssetIO/pull/1308/commits/5a0eb315e512cc119373b53b1d52ceb3275fa39d